adops/gam-hbt-step-by-step.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: page_v2
+title: Header Bidding Trafficking with Prebid Step by Step
+head_title: Header Bidding Trafficking with Prebid Step by Step
+description: Step-by-step instructions for setting up Header Bidding Trafficking in GAM for Prebid.
+#note the sidebar type needs to reflect the section this file is displayed in. See _data/sidenav.yml for the side nav categories.
+sidebarType: 3
+---
+
+# Header Bidding Trafficking with Prebid Step by Step
+
+{: .no_toc }
+
+- TOC
+{: toc }
+
+Publishers new to header bidding can use this guide to set up header bidding trafficking for Prebid in Google Ad Manager. Unlike a line item setup, header bidding trafficking provides publishers with precise bid prices to compete in the auction and no need for shell creatives. It also provides unified reporting, and a straightforward workflow. These instructions describe only the specific settings required for header bidding trafficking; they are not intended to be comprehensive instructions that replace or duplicate the GAM documentation.
```

Comment:
'shell creatives' isn't a term I'm familiar with. Also ,, 'no need' is a bit strong, as some publishers may need custom creative.

Reply:
There used to be limits with HBT -- only worked for display, didn't support aliases, and only certain bidders. Is that still the case or does it now support video, native, aliases?